src/bin/cql-stress-cassandra-stress/operation/mixed.rs

```diff
@@ -111,6 +148,18 @@ impl MixedOperationFactory {
             stats,
         })
     }
+
+    async fn conditional_create_factory<Factory, Fut: Future<Output = Result<Factory>>>(
+        ratios: &OperationRatio,
+        command_kind: &MixedSubcommand,
+        create_factory_fut: impl FnOnce() -> Fut,
+    ) -> Option<Result<Factory>> {
+        if ratios.contains(command_kind) {
+            Some(create_factory_fut().await)
+        } else {
+            None
+        }
+    }
```

Comment:
You still haven't constrained `Factory: CassandraStressOperationFactory`, but nvm.